vars/githubWorkflowRun.txt

```diff
@@ -0,0 +1,113 @@
+Run workflow on github actions
+
+### Run as step:
+
+```
+  Map runInfo = githubWorkflowRun(repo: "owner/repository", workflow: "build.yml", ref: "main",
+    parameters: [path: "filebeat"], credentialsId: "github-workflow-token")
+```
+
+#### Run asyncronous:
+
+```
+  script {
+    Map args = [
+       repo: "owner/repository",
+       workflow: "build.yml", 
+       ref: "main",
+       parameters: [path: "filebeat"],
+       credentialsId: "github-workflow-token"]
+    int runId = githubWorkflowRun.triggerGithubActionsWorkflow(args)
+    Map runInfo = githubWorkflowRun.getWorkflowRun(args + [runId: runId]) 
+  }
+
+```
+
+### Arguments:
```

Comment:
There are some missing arguments to be listed here, such as the `credentialsId`

Reply:
fixed